.github/workflows/build.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: Build
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
Is this needed?

Reply:
The goal was to be able to detect asynchronously whether 3d-party links are broken. Maybe we update this repository often enough that we can omit the cron configuration for now.